examples/cli/src/main/java/io/unitycatalog/cli/ModelVersionCli.java

```diff
@@ -0,0 +1,112 @@
+package io.unitycatalog.cli;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import io.unitycatalog.cli.utils.CliParams;
+import io.unitycatalog.cli.utils.CliUtils;
+import io.unitycatalog.client.ApiClient;
+import io.unitycatalog.client.ApiException;
+import io.unitycatalog.client.api.ModelVersionsApi;
+import io.unitycatalog.client.model.CreateModelVersion;
+import io.unitycatalog.client.model.FinalizeModelVersion;
+import io.unitycatalog.client.model.UpdateModelVersion;
+import org.apache.commons.cli.CommandLine;
+import org.json.JSONObject;
+
+public class ModelVersionCli {
```

Comment:
While it is a sub entity, I'm not sure this makes a ton of sense since model versions are mostly reprsented as full class entities to users via the API and UI.